### PR TITLE
[17.0][MIG] account_comment_template: Migration to 17.0

### PR DESCRIPTION
**Pull request by RogerSans**
supersede: https://github.com/OCA/account-invoice-reporting/pull/317
This PR depends on https://github.com/OCA/reporting-engine/pull/876

Standar migration
T-6435

@HaraldPanten 
@ValentinVinagre

### CODE REVIEW

**Review by luis-ron — APPROVED** [review]
Functional Review: LGTM 👍🏻

**Comment by RogerSans** [comment]
@manuelregidor Con you review this pls?

File: account_comment_template/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# Copyright 2013-2014 Nicolas Bessi (Camptocamp SA)
+# Copyright 2018 Qubiq - Xavier Jiménez
+# Copyright 2018 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Account Comments",
+    "summary": "Comments templates on invoice documents",
+    "version": "17.0.1.0.0",
+    "category": "Sale",
```
Comment (manuelregidor) [inline account_comment_template/__manifest__.py]: Don't you think the category should be accounting?

Comment (RogerSans) [reply account_comment_template/__manifest__.py]: Changed

**Review by manuelregidor — APPROVED** [review]
Technical review. LGTM

**Review by HaraldPanten — APPROVED** [review]

**Pull request merged**